packages/react/src/primitives/types/base.ts

```diff
@@ -19,6 +19,13 @@ export interface BaseComponentProps {
    * Used to provide a `data-testid` attribute for testing purposes
    */
   testId?: string;
+
+  /**
+   * @description
+   * Support for the HTML `inert` attribute
+   * https://developer.mozilla.org/en-US/docs/Web/API/HTMLElement/inert
+   */
+  inert?: string | null;
```

Comment:
Should be boolean?

Reply:
Then you can also `inert: inert ? '' : null,`

Reply:
Yeah I'm wondering if the type as `string | null` might be confusing to developers who aren't familiar with its implementation (i.e. that gh issue I linked in the feature request)?

Reply:
yea that makes sense, updated to boolean